code/modules/admin/STUI.dm

```diff
@@ -65,12 +65,10 @@ GLOBAL_DATUM_INIT(STUI, /datum/STUI, new)
 	.["tabs"] = list()
 	if(user.client.admin_holder.rights & R_MOD)
 		.["tabs"] += STUI_TEXT_ATTACK
-	if(user.client.admin_holder.rights & R_ADMIN)
 		.["tabs"] += STUI_TEXT_ADMIN
```

Comment:
i reckon it'd be better to rename these items to make them less confusing for future coders instead of just changing the perms